src/rules/ex-rule-name.js

```diff
@@ -0,0 +1,28 @@
+CSSLint.addRule({
+
+    // rule information
+    id: "rule-name",
+    name: "Rule name must concat with -",
```

Comment:
The name says it should concatenate with a dash, but the test checks that there's no underscore, so there's a mismatch in the description and the actual behavior of the code. I suggest to change the name and desc.